modules/cnf-image-based-upgrade-seed-image-config.adoc

```diff
@@ -0,0 +1,129 @@
+// Module included in the following assemblies:
+// * edge_computing/image-based-upgrade/cnf-preparing-for-image-based-upgrade.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-image-based-seed-image-config_{context}"]
+= Seed image configuration
+
+The seed image targets a set of {sno} clusters with similar configuration.
+This means that the seed image must have all the components and configuration that the seed cluster shares with the target clusters.
+Therefore, the seed image generated from the seed cluster cannot contain any cluster-specific configuration.
+
+The following table lists the components, resources, and configurations that you must and must not include in your seed image:
+
+.Seed image configuration
+[cols=2*, width="80%", options="header"]
+|====
+|Cluster configuration
+|Include in seed image
+
+|Performance profile
+|Yes
+
+|`MachineConfig` resources for the target cluster
+|Yes
+
+|IP version ^[1]^
+|Yes
+
+|Set of Day 2 Operators, including the {lcao} and the OADP Operator
+|Yes
+
+|Disconnected registry configuration
```

Comment:
The configuration between the seed and target will be diffferent but if the seed is disconnected, the target must be as well